src/schema/contentful/phoenix.js

```diff
@@ -70,12 +72,40 @@ const typeDefs = gql`
     ${entryFields}
   }
 
+  type Page {
+    "This title is used internally to help find this content."
+    internalTitle: String!
+    "Text displayed big at the top of the page."
```

Comment:
I think this can just be
```suggestion
    "The title for this page"
```